docs/device_protocol.rst

```diff
@@ -19,6 +19,8 @@ Device Protocol IDs
 - 15: `MeterBus (MBus) <https://github.com/pyscada/PyScada-MeterBus>`_
 - 16: Generic dummy device
 - 17: `EMS <https://github.com/pyscada/PyScada-EMS>`_
+- 18: `Operations <https://github.com/pyscada/PyScada-Operations>`_
+- 19: `Aggregation <https://github.com/pyscada/PyScada-Operations>`_
```

Comment:
the URL seems to be wrong

Reply:
It is in my [account](https://github.com/pyscada), so the pyscada organization can fork it to make to link exist.
what do you think ?

Reply:
okay, fork is done

both apps have the same Repo so the URL is okay